app/src/main/res/values/dimens.xml

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <dimen name="dimen_text_main_big">30sp</dimen>
+    <dimen name="dimen_title_big">30sp</dimen>
```

Comment:
its better rename and put any reference that it will be use in a text size prpoperty,

Reply:
Done, changed from "dimen_title_big" to "dimen_title_text_size_big"